gwt-ol3-client/src/main/java/ol/source/ImageWmsParams.java

```diff
@@ -41,6 +41,12 @@ public final void setLayers(String... layerNames) {
     @JsProperty(name = "LAYERS")
     public native String getLayers();
 
+    @JsProperty(name = "CQL_FILTER")
```

Comment:
This is a vendor option that isn't a part of the WMS specification. Therefore it shouldn't be in ImageWmsParams. I see two options to solve this issue:

1. Create a class ImageWmsGeoServerParams that extends ImageWmsParams with special params
and/or
2. Extent all parameter types for supporting any additional properties (using JsPropertyMap)

The second options make sense at all and I already planned the implementation.

Reply:
The second option is implemented. See commit https://github.com/TDesjardins/gwt-ol3/commit/0ee7d60c27affe4905e560a989fb6bcc03d55c02